src/utils/evm/test-routes.ts

```diff
@@ -0,0 +1,335 @@
+        for (const destinationDomain of ethereumConfigs) {
+          const destinationProvider = new providers.JsonRpcProvider(rpcEndpoints[destinationDomain.chainId])
+          const loggingData = {
+            resourceId: resource.resourceId,
+            sourceDomainId: network.id,
+            sourceDomainName: network.name,
+            destinationDomainId: destinationDomain.id,
+            destinationDomainName: destinationDomain.name
+          }
+          if (destinationDomain.id == network.id) {
+            continue
+          }
+
+          for (const destinationResource of destinationDomain.resources as unknown as Array<EvmResource>) {
+            if (destinationResource.resourceId === resource.resourceId) {
+              switch (resource.type) {
+                case ResourceType.FUNGIBLE:
+                  const assetTransfer = new EVMAssetTransfer()
+
+                  await assetTransfer.init(
+                    sourceProvider as providers.BaseProvider,
+                    Environment[environment]
+                  )
+
+                  const transfer = await assetTransfer.createFungibleTransfer(
+                    await evmWallet.getAddress(),
+                    destinationDomain.chainId,
+                    await evmWallet.getAddress(),
+                    resource.resourceId,
+                    '500000000000'
+                  )
+
+                  const fee = await assetTransfer.getFee(transfer)
+                  const approvals = await assetTransfer.buildApprovals(
+                    transfer,
+                    fee
+                  )
+
+                  for (const approval of approvals) {
+                    await evmWallet.sendTransaction(
+                      approval as providers.TransactionRequest
+                    )
+                  }
+
+                  const transferTx =
+                    await assetTransfer.buildTransferTransaction(
+                      transfer,
+                      fee
+                    )
+
+                  await evmWallet.sendTransaction(
+                    transferTx as providers.TransactionRequest
+                  )
+
+                  const valueBefore = await fetchTokenAmount(
+                    destinationProvider,
+                    await evmWallet.getAddress(),
+                    destinationResource.address
+                  )
+
+                  functionCalls.push(waitUntilBridgedFungibleEvm(
+                    loggingData,
+                    result,
+                    valueBefore,
+                    await evmWallet.getAddress(),
+                    destinationResource.address,
+                    destinationProvider
+                  ))
+
+                  break
+                case ResourceType.PERMISSIONLESS_GENERIC:
+                  // hack for cronos and base (dont send GMP to cronos and base)
+                  if (network.id > 3 || destinationDomain.id > 3) {
```

Comment:
Let us remove this. Even if something is failing on testnet currently, there is no need to have something like this hardcoded.